packages/cli/lib/make-index.js

```diff
@@ -0,0 +1,109 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+const util = require('util');
+const glob = util.promisify(require('glob'));
+const path = require('path');
+const fs = require('fs');
+const format = require('./format');
+const debug = require('./debug')('make-index');
+
+/**
+ * A utility function to make index files to export artifacts from a folder
+ * for easier imports
+ *
+ * @param {string} dir The directory for which to generate index files.
+ * @param {object} options
+ * options.nested: boolean = true
+ * options.prefix: string | string[] = ''
+ * options.commentHeader:string = SEE BELOW
+ */
+module.exports = async function(dir, options = {}) {
+  // Only exporting typescript files.
+  // TODO: @virkt25 -- May want this to be configurable in the future?
+  const EXTENSION = '.ts';
+
+  // Default Options for Making an exports index file
+  const defaultOptions = {
+    nested: true,
+    prefix: '',
+    commentHeader: `This is an auto-generated file. DO NOT EDIT.`,
+    commentFooter: `This is an auto-generated file. DO NOT EDIT.`,
```

Comment:
Is there any particular reason for using backticks (template literals) instead of regular single-quotes?